src/Lean/Data/HashMap.lean

```diff
@@ -222,6 +222,8 @@ def insertIfNew (m : HashMap α β) (a : α) (b : β) : HashMap α β × Option
 
 instance : GetElem (HashMap α β) α (Option β) fun _ _ => True where
   getElem m k _ := m.find? k
+  getElem? m k := some (m.find? k) -- Not really sure what should go here. We're meant to be returning an `Option (Option β)`.
```

Comment:
Isn't this a sign that the instance doesn't quite make sense here? I'd expect there to be an `Elem` predicate, and that the result of `getElem` would not be an `Option`.

Reply:
(I get the pragmatic reason why we don't have that right now, of course)

Reply:
Yes, @TwoFX will replace this with something more sensible in the big HashMap refactor.